api/v1beta1/kogitoservices.go

```diff
@@ -201,22 +205,23 @@ type KogitoServiceSpec struct {
 	// Additional labels to be added to the Deployment and Pods managed by the operator.
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Additional Deployment Labels"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	DeploymentLabels map[string]string `json:"deploymentLabels,omitempty"`
 
 	// Additional labels to be added to the Service managed by the operator.
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="Additional Service Labels"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
 	ServiceLabels map[string]string `json:"serviceLabels,omitempty"`
 
 	// +optional
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors=true
 	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.displayName="ConfigMap Properties"
-	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:com.tectonic.ui:text"
+	// +operator-sdk:gen-csv:customresourcedefinitions.specDescriptors.x-descriptors="urn:alm:descriptor:io.kubernetes:ConfigMap"
```

Comment:
The UI fetches the CM and shows it in the console?

Reply:
Well, it theory it should. However I wasn't able to reproduce it, asked Vaibhav to try it as well.

Reply:
Well, if you could reproduce, can you share a print screen here in this PR?